sktime/dists_kernels/compose_tab_to_panel.py

```diff
@@ -125,7 +125,7 @@ def get_test_params(cls, parameter_set="default"):
         # importing inside to avoid circular dependencies
         from sktime.dists_kernels import ScipyDist
 
-        return {"transformer": ScipyDist()}
+        return {"transformer": ScipyDist(), "aggfunc_is_symm": [True, False]}
```

Comment:
this is not how this works, you need to return a list of parameter dicts, not a dict of lists

Reply:
Hm why are tests then passing?

Reply:
don't know. Probably because there is no input parameter validation and the checks are "if aggfunc_if_symm = something else something else"